File: 백준/Bronze/28691. 정보보호학부 동아리 소개/정보보호학부 동아리 소개.py
N = input()
# club = ['MatKor','WiCys','CyKor','AlKor','$clear']


# matching = [s for s in club if N in s] 
# print(matching)
if N == 'M':
    print('MatKor')
elif N == 'W':
    print('WiCys')
elif N == 'C':
    print('CyKor')
elif N == 'A':
    print('AlKor')
elif N == '$':
    print('$clear')

File: 백준/Bronze/28691. 정보보호학부 동아리 소개/정보보호학부 동아리 소개.cc
#include <iostream>

using namespace std;

int main()
{
    string N;
    cin >> N;

    if(N == "M")
    {
        cout << "MatKor" << endl;
    } 
    else if(N == "W")
    {
        cout << "WiCys" << endl;
    }
    else if(N == "C")
    {
        cout << "CyKor" << endl;
    }
    else if(N == "A")
    {
        cout << "AlKor" << endl;
    }
    else if(N == "$")
    {
        cout << "$clear" << endl;
    }
}